discovery-infra/test_infra/helper_classes/config/nodes_config.py

```diff
@@ -32,7 +32,6 @@ class BaseTerraformConfig(_BaseConfig, ABC):
     storage_pool_path: str = None
     # running: bool = True
     single_node_ip: str = None
-    olm_operators: List[str] = None
```

Comment:
Have you checked to make sure it's not being used elsewhere (test_infra, QE)?

Reply:
I will double check

Reply:
According to my findings this is only used from `ClusterConfig`